src/java/picard/analysis/CollectWgsMetricsFromSampledSites.java

```diff
@@ -0,0 +1,44 @@
+package picard.analysis;
+
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.util.*;
+import picard.cmdline.CommandLineProgramProperties;
+import picard.cmdline.Option;
+import picard.cmdline.programgroups.Metrics;
+
+import java.io.File;
+
+/**
+ * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments,
+ * but only at a set of sampled positions.
+ *
+ * @author ebanks
+ */
+@CommandLineProgramProperties(
+        usage = "Computes a number of metrics that are useful for evaluating coverage and performance of " +
+                "whole genome sequencing experiments, but only at a set of sampled positions.",
+        usageShort = "Writes whole genome sequencing-related metrics for a SAM or BAM file",
+        programGroup = Metrics.class
+)
+public class CollectWgsMetricsFromSampledSites extends CollectWgsMetrics {
+
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the locations of the positions to assess.", optional = false)
+    public File INTERVALS = null;
+
+
+    public static void main(final String[] args) {
+        new CollectWgsMetricsFromSampledSites().instanceMainWithExit(args);
+    }
+
```

Comment:
I don't think that check is helpful.  What if you sample every other base?  That's still not desired behavior but it would pass this check.
I'll add very clear docs up top.